### PR TITLE
Added controls for car, bench, garden, tv. Updated drawable.xaml aswell.

### PR DESCRIPTION
**Pull request by cbabrunal**
Stylecop checked as well. I used Garden instead of Tree this time around.

### CODE REVIEW

File: HiVRClient/View/DrawableTemplate.xaml [hunk 1]
```diff
@@ -28,4 +28,115 @@
...
+                </TransformGroup>
+            </Path.RenderTransform>
+        </Path>
+    </DataTemplate>
+    <DataTemplate
+        DataType="{x:Type viewmodel:TelevisionControl}">
+        <Path>
+            <Path.Data>
+                <RectangleGeometry
+                    Rect="0,10,20,0" />
```
Comment (wfdewith) [inline HiVRClient/View/DrawableTemplate.xaml]: I think this isn't correct.

**Comment by keohnoel** [comment]
I think so to Wim. I fixed it.

**Comment by wfdewith** [comment]
LGTM

**Comment by wfdewith** [comment]
Technically you're not allowed to do this, but whatever

**Pull request merged**